src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -541,6 +551,13 @@ private bool IsAtStartOfEmbeddedExpression() =>
             CurrentToken is { Text: "{" } &&
             CurrentTokenPlus(1) is { Text: "{" };
 
+        private bool IsStartOfEscapedCharacterSequence() =>
+            CurrentToken is { Kind: TokenKind.Punctuation } and { Text: "\\" } &&
```

Comment:
Speaking of escaping, I think this read a little more easily:

```suggestion
            CurrentToken is { Kind: TokenKind.Punctuation } and { Text: @"\" } &&
```